data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1234 @@
+				to display
+					or
+						"reputation: Gegno Vi" < 0
+						"reputation: Gegno Scin" < 0
+
+			`	The bulkiest Gegno of the trio then places a model of a massive Vi warship in the middle of the table, one that you definitely recognize. More angry outbursts come from the surrounding Gegno, and while the one who placed the model glares at you, the other two eventually deescalate the uproar.`
+				to display
+					has "dead augen"
+
+
+			`	After having gone through all of the cases which involved you, the trio addresses the gathering once more with a big speech. Their voices echo through the outside air, and all of the Gegno respond with concurrent bellows or chants, followed by more silence. Oddly enough, you feel as if you are standing among them as part of the gathering, rather than as some foreign alien. You aren't restricted at all, even with a weapon on you.`
+			`	One of the smaller Gegno in the trio gestures back to you and speaks once more. You still don't understand a word they are saying, but it must have been important, as all of the other Gegno except your guide back away several steps. The trio member invites you with both arms to use the objects and the table, while all three of them back up themselves, leaving you alone in the open.`
+			choice
+				`	(Approach the table and try to explain the incidents you were involved in.)`
+					goto "explain"
+				`	(Stay where you are and do nothing.)`
+
+			action
+				set "gegno: no proclamation"
+			`	You don't approach the table, standing awkwardly in the open. Maybe it's better to not say anything at all if you're what they perceive as the problem here. A whole minute of silence goes by, and all of the Gegno, including the trio, haven't said so much as a word. Instead, they silently judge the choice you've decided to make. The trio then march back to the table, moving all of the models aside as if they're setting up for something.`
+				goto "what does home mean"
+
+
+			label "explain"
+			`	Walking up to the table, you try your hardest to explain each situation one by one. Since you can't read or do anything with the actual display on the monitor, you try picking up the models, which nobody seems to be bothered by. At the same time, you gesture using your hands, mainly representing your ship flying "away". You wonder if anything makes any sense to the Gegno, but they don't seem to be paying attention to your words or actions themselves; rather, they seem to respond to the tone and emotion of them. After doing your best to describe everything, you back away from the table. The trio march back up to it, discussing your response amongst themselves. They start moving all of the models aside as if they're setting up for something.`
```

Comment:
```suggestion
			`	Walking up to the table, you try your hardest to explain each situation one by one. Since you can't read or do anything with the actual display on the monitor, you try picking up the models, which nobody seems to be bothered by. At the same time, you gesture using your hands, mainly representing your ship flying "away". You wonder if anything makes any sense to the Gegno, but they don't seem to be paying attention to your words or actions themselves; rather, they seem to respond to the tone and emotion of them. After doing your best to describe everything, you back away from the table. The trio march back up to it, discussing your response amongst themselves. They then start moving all of the models aside as if they're setting up for something.`
```